webapp/components/expression/expressionEditorPopup.js

```diff
@@ -55,24 +58,25 @@ const ExpressionEditorPopup = (props) => {
   const i18n = useI18n()
 
   return (
-    <PanelRight onClose={onClose} width="1020px">
+    <PanelRight onClose={onClose} width="1020px" header={header}>
       <div className="expression-editor-popup">
-        <button
-          type="button"
-          className="expression-editor-popup__toggle-advanced btn-s"
-          hidden={hideAdvanced}
-          onClick={onToggleAdvancedEditor}
-        >
-          {advanced ? i18n.t('nodeDefEdit.basic') : i18n.t('nodeDefEdit.advanced')}
-        </button>
-
+        {type.includes(ExpressionEditorType.basic) && type.includes(ExpressionEditorType.advanced) && (
```

Comment:
you can shorten this by [ExpressionEditorType.advanced, ExpressionEditorType.basic].includes(type)

Reply:
"type" is an array, it's name can be misleading, should it be called "types" instead?

Reply:
yep i felt that type is an string